albatross.ml

```diff
@@ -38,6 +48,32 @@ struct
     in
     Ok (Vmm_trie.fold path t.policies (fun name p acc -> (name, p) :: acc) [])
 
+  let policy_resource_used t =
```

Comment:
the function name is misleading, it returns the unassigned resources, not the used ones (at least for vms, memory, block).